From 912ff9ad25ff8f4731336b450d9201ed8999464c Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Tue, 17 Jul 2018 15:04:28 +0200
Subject: [PATCH] wget: Deal with optional libpsl dependency

Enable dependency in full wget and add it to dependencies and drop it in no-ssl
variant. If you are not using ssl, you are probably heavily size constraint, so
you probably don't care about this either.

Signed-off-by: Michal Hrusecky <michal.hrusecky@nic.cz>
---
 net/wget/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/net/wget/Makefile b/net/wget/Makefile
index 64a6a08..0fbc630 100644
--- a/net/wget/Makefile
+++ b/net/wget/Makefile
@@ -42,7 +42,7 @@ endef
 
 define Package/wget
 $(call Package/wget/Default)
-  DEPENDS+= +libopenssl +librt
+  DEPENDS+= +libopenssl +librt +libpsl
   TITLE+= (with SSL support)
   VARIANT:=ssl
 endef
@@ -81,7 +81,8 @@ endif
 ifeq ($(BUILD_VARIANT),nossl)
 	CONFIGURE_ARGS+= \
 		--disable-ntlm \
-		--without-ssl
+		--without-ssl \
+		--without-libpsl
 endif
 
 define Package/wget/install
-- 
2.18.0

